apps/workflows/src/cron/emails.ts

```diff
@@ -14,16 +14,33 @@ export async function sendFollowUpEmails() {
   date2.setDate(date2.getDate() - 2);
 
   const users = await db
-    .select()
+    .select({
+      email: user.email,
+    })
     .from(user)
     .where(and(gte(user.createdAt, date1), lte(user.createdAt, date2)))
     .all();
 
   console.log(`Found ${users.length} users to send follow ups.`);
 
-  for (const user of users) {
-    if (user.email) {
-      await email.sendFollowUp({ to: user.email });
+  // Filter valid emails
+  const validEmails = users
+    .map((u) => u.email)
+    .filter((email) => email !== null)
+    //  I don't know why but I can't have both filter at the same time
+    .filter((email) => typeof email === "string" && email.trim() !== "");
```

Comment:
![CleanShot 2025-05-01 at 12 01 49@2x](https://github.com/user-attachments/assets/77c89dbc-b5fd-4a64-b145-1c25950e18ec)

I dont understand the typeof email here, because the type is already infered ?